kws/model/mdtc.py

```diff
@@ -187,6 +187,9 @@ def __init__(
     ):
         super(MDTC, self).__init__()
         self.kernel_size = kernel_size
+        if kernel_size % 2 == 0:
```

Comment:
move it before self.kernel_size = kernel_size
use  `assert kernel_size % 2 == 0`

Reply:
done